src/status_im/ui/screens/events.cljs

```diff
@@ -401,8 +407,13 @@
 
 (handlers/register-handler-fx
   :app-state-change
-  (fn [_ [_ state]]
-    (status/app-state-change state)))
+  (fn [{{:keys [network-status mailserver-status]} :db :as cofx} [_ state]]
+    (let [should-recover? (and (= state "active")
```

Comment:
could you add a comment indicating that this is where we know that the app is coming from background ?

Reply:
moved condition to new let binding with meaningful name